src/main/java/seedu/address/commons/core/Commands.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.commons.core;
+
+import seedu.address.logic.commands.AddCommand;
+import seedu.address.logic.commands.ClearCommand;
+import seedu.address.logic.commands.ClearHistoryCommand;
+import seedu.address.logic.commands.DeleteCommand;
+import seedu.address.logic.commands.EditCommand;
+import seedu.address.logic.commands.ExitCommand;
+import seedu.address.logic.commands.FindCommand;
+import seedu.address.logic.commands.HelpCommand;
+import seedu.address.logic.commands.HistoryCommand;
+import seedu.address.logic.commands.ListCommand;
+import seedu.address.logic.commands.RedoCommand;
+import seedu.address.logic.commands.SelectCommand;
+import seedu.address.logic.commands.UndoCommand;
+
+/**
+ * Container for all command words and command aliases in the application.
+ */
+public class Commands {
+    public static String[] getAllCommandWords() {
+        String[] allCommandWords =  new String[] {
```

Comment:
I would recommend creating these arrays as a static field, preferably wrapped around a `List` as in `Arrays.asList`, unless there is a technical requirement to expose *only* a `String` array. This will prevent the array being created from scratch on every function call, which might become expensive as I anticipate autocomplete to call this many times.

Reply:
Great suggestion, thanks!